sources/api/netdog/src/wicked/mod.rs

```diff
@@ -3,16 +3,28 @@
 //!
 //! The structures in this module are meant to be created from the user-facing structures in the
 //! `net_config` module.  `Default` implementations for WickedInterface exist here as well.
+mod bonding;
 mod dhcp;
 mod static_address;
+mod vlan;
 
 use crate::interface_name::InterfaceName;
+use crate::net_config::devices::bonding::{BondMonitoringConfig, NetBondV1};
+use crate::net_config::devices::interface::NetInterfaceV2;
+use crate::net_config::devices::vlan::NetVlanV1;
+use crate::net_config::devices::NetworkDeviceV1;
+use crate::wicked::bonding::{
+    WickedArpMonitoringConfig, WickedBondMode, WickedMiiMonitoringConfig,
+};
+pub(crate) use bonding::WickedBond;
```

Comment:
Nit: I might have missed it, but does this need to be `pub`?

Reply:
It looks like this one no longer needs to be `pub`. I'll fix it.